src/docfx/build/page/BuildPage.cs

```diff
@@ -21,7 +21,8 @@ internal static class BuildPage
             Debug.Assert(file.ContentType == ContentType.Page);
 
             var (errors, schema, model, yamlHeader) = await Load(context, file, callback);
-            var (metaErrors, metadata) = JsonUtility.ToObject<FileMetadata>(file.Docset.Metadata.GetMetadata(file, yamlHeader));
+            var temp = file.Docset.Metadata.GetMetadata(file, yamlHeader);
```

Comment:
maybe `GetMetadata` returns `FileMetadata`?

Reply:
was using it for debugging, removed